data/mods/gen1/moves.ts

```diff
@@ -268,30 +268,39 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		},
 	},
 	disable: {
-		inherit: true,
+		num: 50,
+		accuracy: 55,
+		basePower: 0,
+		category: "Status",
+		name: "Disable",
+		pp: 20,
+		priority: 0,
+		flags: {protect: 1, mirror: 1, bypasssub: 1},
+		volatileStatus: 'disable',
+		onTryHit(target) {
+			return target.moveSlots.some(ms => ms.pp > 0) && undefined;
```

Comment:
This shouldn't need && undefined.

Reply:
You're missing a check to see if one of the opponent's moves are already Disabled as well (right now it will just silently proceed).